src/main/java/Problem3/MovieAction.java

```diff
@@ -22,5 +30,14 @@ public int getLateFeeInDollar() {
     @Override
     public int calcLateFees(int numOfDaysPastDue) {
         // homework
+        if (numOfDaysPastDue < 0) {
+            numOfDaysPastDue = 0;
+        }
+        if(numOfDaysPastDue < 5) {
+            return numOfDaysPastDue * lateFeePerDayInDollar;
+        }
+        else {
+            return 2 * numOfDaysPastDue * lateFeePerDayInDollar;
+        }
```

Comment:
duplicated logic here. 

suggesting:
```java
        int fee = lateFeePerDayInDollar * numOfDaysPastDue;
        return (numOfDaysPastDue >= 5) ? 2 * fee : fee;
```

and here number 5 only appears once too